mlir/lib/Dialect/XeGPU/IR/XeGPUOps.cpp

```diff
@@ -38,6 +41,38 @@ static std::string makeString(T array, bool breakline = false) {
   return buf;
 }
 
+static std::vector<int64_t> getShapeOf(Type type) {
```

Comment:
```suggestion
static SmallVector<int64_t> getShapeOf(Type type) {
```

(`SmallVector` is the usual default)